app/src/main/java/com/imaginaryrhombus/proctimer/ui/timer/TimerFragment.kt

```diff
@@ -115,6 +148,17 @@ class TimerFragment : Fragment() {
 
         viewModel.isTimerWorking.observe(this, Observer {
             setKeepScreenOn(it)
+
+            if (it) {
+                serviceIntent = Intent(requireContext(), TimerService::class.java)
+                requireContext().bindService(serviceIntent, serviceConnection, Context.BIND_AUTO_CREATE)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fizxive/ProcTimer/blob/6398337ef2821758c0dc8bf3d021690793d3fc76/app/src/main/java/com/imaginaryrhombus/proctimer/ui/timer/TimerFragment.kt#L154"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>